ts/ui/menu/MmlVisitor.ts

```diff
@@ -97,4 +100,22 @@ export class MmlVisitor<N, T, D> extends SerializedMmlVisitor {
                  + space + '</math>';
   }
 
+  /**
+   * @override
+   */
+  protected getAttributeList(node: MmlNode): PropertyList {
+    const list = super.getAttributeList(node);
+    if (this.options.filterTex) {
+      delete list['data-latex'], list['data-latexItem'];
+    }
+    if (this.options.filterSRE) {
+      const keys = Object.keys(list).filter(
+        id => id.match(/^(?:data-semantic-.*?|role|aria-(?:level|posinset|setsize))$/)
```

Comment:
What if we parse from MathML and some of these aria attributes are already in the incoming element?
(I suppose we would have overwritten them at this point anyway.)

Reply:
This filtering only occurs in the `Show Math As` or `Copy Math As` menus.  Nothing is changed internally.  This filtering is controlled by menu options, so the user can turn off the filtering in the case where retaining those values are desired.  Does that put you more at ease?

Reply:
Note that this `MmlVisitor` is in the `ts/ui/menu` directory, not the one in `ts/core/MmlTree`, so it only affects the menu output.

Reply:
Thanks. Sounds good.